hca/dss/__init__.py

```diff
@@ -610,3 +611,107 @@ def _parse_manifest(cls, manifest):
             delimiter = b'\t' if USING_PYTHON2 else '\t'
             reader = csv.DictReader(f, delimiter=delimiter, quoting=csv.QUOTE_NONE)
             return reader.fieldnames, list(reader)
+
+    def _serialize_col_to_manifest(self, uuid, replica, version, _max_depth=5,
+                                   _ignore=[]):
+        """
+        Given a collection UUID, uses GET `/collection/{uuid}` to
+        serialize the collection into a set of dicts that that can be
+        used to generate a manifest file.
+
+        Most of the heavy lifting is handled by
+        :meth:`DSSClient.download_manifest`.
+
+        :param uuid: uuid of the collection to serialize
+        :param replica: replica to query against
+        :param version: version of the specified collection
+        :param int _max_depth: maximum depth to serialize - used to limit
```

Comment:
Depth shouldn't matter or be a limiting factor here.  Consider using a while loop instead of recursion.